applications/reference-data/src/test/java/no/dcat/themes/builders/LosThemeTest.java

```diff
@@ -19,8 +17,7 @@
 
 @SpringBootTest
 @RunWith(SpringRunner.class)
-@Category(UnitTest.class)
-@ActiveProfiles(profiles = "unit-integration")
+@Category(IntegrationTest.class)
```

Comment:
Jeg ser at det er flere committer som har rød mark fra jenkins. Så det ser ut at den bransj har vært ustabilt en stund til det får fikset til slutten. 

Jeg anbefaler at i develop bransj, vi har hver eneste commit stabil.

Det er to løsning:
1) gå tilbake til den kommit som brytet test, og inkludere testfiks der.
eller
2) squash alt sammen i en svært stort kommit

Reply:
blir en stor commit